citations-tool/tool/src/webapp/vm/citation/create.vm

```diff
@@ -376,14 +376,15 @@
 	 */
 	function trackPreferredUrl( checkboxId )
 	{
-		// has this been clicked?
-		if( $( "#" + checkboxId ).attr( "checked" ) != null )
+		var checkBoxClicked = $( "#" + checkboxId );
+		// has this been checked?
+		if( checkBoxClicked.attr( "checked" ) == true )
```

Comment:
But what does it return 1.6+ as it may well be upgrade shortly. I think the `jQuery way` of doing this is with the `checkBoxClicked.is(":checked")`.